zap/src/main/java/org/parosproxy/paros/core/scanner/AppParamValueType.java

```diff
@@ -0,0 +1,13 @@
+package org.parosproxy.paros.core.scanner;
+
+/**
+ * {@code AppParamValueType} represents the type of AppParameter's value. This is useful for the
+ * use-cases like say there is a parameter which is a URL then caller can itself encode/escape it so
+ * that it can be directly used to modify {@code HttpMessage} or {@code Variant} has to escape it.
+ *
+ * @author preetkaran20@gmail.com KSASAN
+ */
+public enum AppParamValueType {
```

Comment:
Better move to `AppParameter`, this also applies to the name not just the value.

Reply:
ok, `with the comment this also applies to name`, you are suggesting for javadoc?

Reply:
Was about the name of the enum (though JavaDoc should be updated accordingly), e.g. `PayloadFormat`?

Reply:
ok